### PR TITLE
Rework email descriptions

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request reworks email descriptions, and related files, to conform to the [writing style](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md).

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/EmailMessages/EmailMessage.yaml [hunk 1]
```diff
@@ -89,23 +90,23 @@ properties:
         - resourceId
       properties:
         resourceType:
-          description: The attachment's resource type.
+          description: Resource type of the attachment.
```
Comment (n-fasano) [inline openapi/components/schemas/EmailMessages/EmailMessage.yaml:92]:
```suggestion
          description: Type of attachment resource.
```

File: openapi/components/schemas/EmailMessages/EmailMessage.yaml [hunk 2]
```diff
@@ -67,20 +68,20 @@ properties:
...
     maxLength: 998
     example: Welcome dear customer!
   text:
-    description: The email message's text body.
+    description: Text body of the email message.
     type: string
     example: Email message text body
   html:
-    description: The email message's html body.
+    description: HTML body of the email message.
```
Comment (n-fasano) [inline openapi/components/schemas/EmailMessages/EmailMessage.yaml:79]:
```suggestion
    description: Body of the email message in HTML format.
```

File: openapi/components/schemas/EmailNotifications/EmailNotification.yaml [hunk 3]
```diff
@@ -1,33 +1,33 @@
...
     type: array
     minItems: 1
-    description: The list of notifications.
+    description: List of notifications.
     items:
       type: object
       properties:
         labels:
-          description: The notification labels.
+          description: Notification label.
```
Comment (n-fasano) [inline openapi/components/schemas/EmailNotifications/EmailNotification.yaml:20]:
```suggestion
          description: Label of the notification.
```

File: openapi/components/schemas/EmailNotifications/EmailNotification.yaml [hunk 4]
```diff
@@ -1,33 +1,33 @@
...
     type: array
     minItems: 1
-    description: The list of notifications.
+    description: List of notifications.
     items:
       type: object
       properties:
         labels:
-          description: The notification labels.
+          description: Label of the notification.
```
Comment (ashkarpetin) [inline openapi/components/schemas/EmailNotifications/EmailNotification.yaml:20]:
```suggestion
          description: Labels of the notification.
```

File: openapi/components/schemas/EmailNotifications/SendThroughAttribution.yaml [hunk 5]
```diff
@@ -1,22 +1,22 @@
...
   sent:
     type: integer
     minimum: 0
-    description: The email notification send number.
+    description: Number of email notifications that have been sent.
   goal:
     type: integer
     minimum: 0
-    description: The email notification goal number.
+    description: Ideal, or target, number of email notifications to send.
```
Comment (ashkarpetin) [inline openapi/components/schemas/EmailNotifications/SendThroughAttribution.yaml:22]: This looks confusing

Comment (noon-dawg) [reply openapi/components/schemas/EmailNotifications/SendThroughAttribution.yaml:22]: How about one of the following:
- Target number number of email notifications to send.
- Intended number of email notifications to send.

Comment (n-fasano) [reply openapi/components/schemas/EmailNotifications/SendThroughAttribution.yaml:22]: `Target number of email notifications to send.`

I think this one is best.

Comment (noon-dawg) [reply openapi/components/schemas/EmailNotifications/SendThroughAttribution.yaml]: Thanks guys

**Review by ashkarpetin — DISMISSED** [review]

**Review by n-fasano — DISMISSED** [review]

File: openapi/components/schemas/EmailNotifications/SendThroughAttribution.yaml [hunk 6]
```diff
@@ -1,22 +1,22 @@
...
   sent:
     type: integer
     minimum: 0
-    description: The email notification send number.
+    description: Number of email notifications that have been sent.
   goal:
     type: integer
     minimum: 0
-    description: The email notification goal number.
+    description: Target number number of email notifications to send.
```
Comment (n-fasano) [inline openapi/components/schemas/EmailNotifications/SendThroughAttribution.yaml:22]:
```suggestion
    description: Target number of email notifications to send.
```

**Review by n-fasano — APPROVED** [review]

**Review by adamaltman — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**